simtools/simtel/simtel_runner_array.py

```diff
@@ -1,7 +1,7 @@
 import logging
 from pathlib import Path
 
-import simtools.io_handler as io
+import simtools.io_handler as io_handler
```

Comment:
Can't you simply do `from simtools import io_handler` instead? It's obviously not important, but it is a little strange (appears in other places as well).

Reply:
Fixed (and this is one of the weird things for me in python).